### PR TITLE
Replace nvbw images with CC0 images

### PR DESCRIPTION
**Pull request by wielandb**
This PR will replace all images in the directory /dependencies/nvbw_open_data/images with images that I took myself and put under the CC0 license.

### CODE REVIEW

**Comment by wielandb** [comment]
At this point, 18 images remain to be replaced.

**Comment by wielandb** [comment]
At this point, 17 images remain to be replaced

**Comment by wielandb** [comment]
At this point, 4 images remain to be replaced.

File: assets/images/questions/1051b.jpg
Comment (Robbendebiene) [inline assets/images/questions/1051b.jpg:1]: If possible an image with less surroundings would be preferred. Alternatively you can crop the image if the resolution allows it.

File: assets/images/questions/1051c.jpg
Comment (Robbendebiene) [inline assets/images/questions/1051c.jpg:1]: An image similar to this without a car in the front (instead maybe a parking sign) would be preferable.

File: assets/images/questions/1051d.jpg
Comment (Robbendebiene) [inline assets/images/questions/1051d.jpg:1]: Please align the image

File: assets/images/questions/1051g.jpg
Comment (Robbendebiene) [inline assets/images/questions/1051g.jpg:1]: Crop image left & right or use a similar image in portrait mode.

File: assets/images/questions/1052d.jpg
Comment (Robbendebiene) [inline assets/images/questions/1052d.jpg:1]: Can be deleted since we are not using it (yet)

File: assets/images/questions/1120a.jpg
Comment (Robbendebiene) [inline assets/images/questions/1120a.jpg:1]: We support the idea, but the big blurred chunk of people doesn't look right to us. Please try to find an alternative.

File: assets/datasets/question_catalog.json [hunk 1]
```diff
@@ -389,7 +389,7 @@
          "text": "Gibt es an der Haltestelle Sitzplätze?",
          "image": [
             "packages/nvbw_open_data/photos/1120a.jpg",
-            "packages/nvbw_open_data/photos/1120b.jpg",
+            "assets/images/questions/1120b.jpg",
             "packages/nvbw_open_data/photos/1120c.jpg",
```
Comment (Robbendebiene) [inline assets/datasets/question_catalog.json:393]: Reuse `assets/images/questions/1120c.jpg`

File: assets/images/questions/1140a.jpg
Comment (Robbendebiene) [inline assets/images/questions/1140a.jpg:1]: Please align image horizontally.

**Review by Robbendebiene — CHANGES_REQUESTED** [review]
- [ ] Please change all image names to a meaningful name like: "parking_surface" "parking_wheelchair_1".
- [ ] Generally reduce the saturation of the images.
- [ ] Ensure minimal resolution of 1000x1000 and a maximal resolution of 2000x2000

File: assets/images/questions/1120a.jpg
Comment (wielandb) [reply assets/images/questions/1120a.jpg:1]: Would retaking the photo without people in it be sufficient?

File: assets/images/questions/1051d.jpg
Comment (wielandb) [reply assets/images/questions/1051d.jpg:1]: I am not really sure what is meant by that.

File: assets/images/questions/parking_street_side.jpg
Comment (wielandb) [inline assets/images/questions/parking_street_side.jpg:1]: Crop bottom

File: assets/images/questions/parking_multistorey_2.jpg
Comment (wielandb) [inline assets/images/questions/parking_multistorey_2.jpg:1]: Use this image for multi storey

File: assets/images/questions/handrail_2.jpg
Comment (wielandb) [inline assets/images/questions/handrail_2.jpg:1]: Reduce saturation/contrast further

File: assets/images/questions/IMG_20230505_125425.jpg
Comment (wielandb) [inline assets/images/questions/IMG_20230505_125425.jpg:1]: Remove

File: assets/images/questions/park_and_ride.jpg
Comment (wielandb) [inline assets/images/questions/park_and_ride.jpg:1]: Reduce saturation and contrast

File: assets/datasets/question_catalog.json [hunk 2]
```diff
@@ -157,9 +157,10 @@
          "name": "Behindertenparkplätze",
          "text": "Gibt es gesonderte Parkplätze für behinderte Personen?",
          "image": [
-            "packages/nvbw_open_data/photos/1051e.jpg",
-            "packages/nvbw_open_data/photos/1051f.jpg",
-            "packages/nvbw_open_data/photos/1051g.jpg"
+			"assets/images/questions/traffic_sign_314_1044-10.png",
```
Comment (Robbendebiene) [inline assets/datasets/question_catalog.json:160]: Please fix indention (convert tabs to spaces)

File: assets/datasets/question_catalog.json [hunk 3]
```diff
@@ -907,7 +903,7 @@
          "text": "Gibt es eine bewegbare Rollstuhl-Rampe?",
          "description": "Bahnsteiggebundene Rampen sind bewegbar und helfen beim Ein- und Ausstieg von Rollstuhlfahrenden zur Überwindung eines Spalts bzw. von Stufen.",
          "image": [
-            "assets/images/questions/1210a.jpg"
+            "assets/images/questions/wheelchair_lift_1.jpg"
```
Comment (7h30n3) [inline assets/datasets/question_catalog.json:906]: A picture of a lift shouldn't be attached to a question about a ramp.

File: pubspec.yaml [hunk 4]
```diff
@@ -46,9 +46,6 @@ dependencies:
   flutter_web_auth: ^0.5.0
   material_design_icons_flutter: ^6.0.7096
   flutter_markdown: ^0.6.14
-  nvbw_open_data:
-    path: ./dependencies/nvbw_open_data
-
```
Comment (7h30n3) [inline pubspec.yaml:51]: Please restore the empty line.

File: assets/datasets/question_catalog.json (hunk 3 above)
Comment (wielandb) [reply assets/datasets/question_catalog.json]: Renamed image to wheelchair_ramp_1 for more clairity

**Comment by wielandb** [comment]
> * [ ]  Ensure minimal resolution of 1000x1000 and a maximal resolution of 2000x2000

At the moment, the following images violate this rule:

- [ ] approach_aid_no.jpg (too big!)
- [ ] automatic_door_button.jpg (too big!)
- [ ] disabled_parking_1.jpg (too small!)
- [ ] disabled_parking_2.jpg (too small!)
- [ ] disabled_parking_3.jpg (too small!)
- [ ] handrail_2.jpg (too small!)
- [ ] handrail_tactile_writing_1.jpg (too big!)
- [ ] handrail_tactile_writing_2.jpg (too big!)
- [ ] kerb_flush.jpg (too big!)

**Comment by 7h30n3** [comment]
The only two things missing are a new image for streetside parking and checking if all images are numbered correctly.

**Comment by wielandb** [comment]
Should be ready for final review now. :)

**Pull request merged**